ibis_substrait/compiler/translate.py

```diff
@@ -480,6 +482,11 @@ def value_op(
             function_reference=compiler.function_id(expr),
             output_type=translate(expr.type()),
             arguments=[
+                stalg.FunctionArgument(
```

Comment:
This option doesn't apply to all scalar functions, which I think is what's causing [the bulk of the CI failures](https://github.com/ibis-project/ibis-substrait/actions/runs/3268859832/jobs/5375755332#step:6:1203). I'm not sure what the best ibis::way is to detect what enum args are supported by a function

Reply:
doing this "correctly" is going to have to wait until we're properly parsing the extension yaml files to read in the options (or until the upstream PR making optional parameters _optional_ lands).

For now this is only impacting decompilation, so I'm going to xfail them and circle back.

Reply:
@jvanstraten -- that is definitely the better target state.  For the moment, I'm after some small degree of interoperability with pyarrow and setting it to `unspecified` doesn't seem to play nice.

Reply:
Huh, based on the current Acero code I thought it should have. Carry on then :)